apps/mobile-alphabet/src/app/Components/Home.tsx

```diff
@@ -1,46 +1,21 @@
-import styles, { colors } from 'app/styles';
+import styles, { alphabetButton, creditsButton } from 'app/styles';
 import React from 'react';
 import { Pressable, Text, View } from 'react-native';
 
 const HomeScreen = ({ navigation }) => {
     return (
-        <View testID="Home" style={styles.background}>
+        <View testID="Home" style={[styles.background]}>
             <Pressable testID="Menu" onPress={() => navigation.push('Menu')}>
-                <Text
-                    style={{
-                        fontSize: 60,
-                        color: colors.primary,
-                        fontWeight: '900',
-                        fontFamily: 'OpenSans',
-                    }}
-                >
-                    Tŝilhqot’in
-                </Text>
-                <Text style={{ fontSize: 40, color: colors.accent, fontFamily: 'OpenSans' }}>
-                    Alphabet{' '}
-                </Text>
+                <Text style={alphabetButton.alphabetName}>Tŝilhqot’in</Text>
```

Comment:
Oh shouldn't this come from the alphabet data via redux? We don't want to hard-wire this.